filebrowser/main.tf

```diff
@@ -78,7 +94,7 @@ resource "coder_app" "filebrowser" {
   display_name = "File Browser"
   url          = "http://localhost:${var.port}"
```

Comment:
Don't we need to set the URL to a correctly formatted URL with a subpath here?

Reply:
The Filebrowser app will set the given basepath if called with the root path (/). So there is no need to set the subpath beforehand.

Reply:
The filebrowser app automatically redirects to the given basepath when called with the root path (/). So there is no need to change it here.
Also this is the correct url if called with subdomain = true.

Reply:
As I understand, the subpath is stripped by the proxy, so the filebrowser app just sees the path behind the SERVER_BASE_PATH. So there is no need to change it.